src/harmony/Badge/Badge.module.css

```diff
@@ -1,24 +1,28 @@
 .Badge {
     display: inline-flex;
-    align-items: center;
-    justify-content: center;
+    align-items: baseline;
+    justify-content: flex-start;
 }
 
 .BadgeOutlined {
     border: 1px solid currentColor;
     border-radius: var(--radius-xl);
 }
 
-.BadgeIconLeft {
-    margin-right: var(--gap-s);
-    display: inline-flex;
+.BadgeIcon {
+    display: inline-block;
     line-height: 1;
+    width: 1em;
```

Comment:
https://github.com/taskany-inc/icons/blob/main/src/components/BaseIcon.tsx#L54

same style on BaseIcon. Why we need copy here?

Reply:
'cause `<Dot />` haven't `BaseIcon` wrapper

without these styles Badge with dot or icon will be looks are different
<img width="88" alt="image" src="https://github.com/taskany-inc/bricks/assets/7002692/7be8774d-f27d-48f4-a557-a48c4309a72a">